client/deepspeech_client.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python2.7
+
+"""A client that talks to tensorflow_model_server loaded with deepspeech model.
+
+The client queries the service with the given audio and prints a ranked list
+of decoded outputs to the standard output, one per line.
+
+Typical usage example:
+
+    deepspeech_client.py --server=localhost:9000 --file audio.wav
+"""
+
+import os
+import sys
+import threading
+from grpc.beta import implementations
+import numpy as np
+import tensorflow as tf
+
+from tensorflow_serving.apis import predict_pb2
+from tensorflow_serving.apis import prediction_service_pb2
+
+sys.path.append(os.path.join(os.path.dirname(os.path.realpath(__file__)), '..'))
+from util.text import ndarray_to_text
+from util.audio import audiofile_to_input_vector
+
+tf.app.flags.DEFINE_string('server', '', 'PredictionService host:port')
+tf.app.flags.DEFINE_string('file', '', 'Wave audio file')
+# These need to match the constants used when training the deepspeech model
+tf.app.flags.DEFINE_string('n_input', 26, 'Number of MFCC features')
+tf.app.flags.DEFINE_string('n_context', 9, 'Number of frames of context')
+FLAGS = tf.app.flags.FLAGS
+
+def _create_rpc_callback(event):
+    def _callback(result_future):
+        exception = result_future.exception()
+        if exception:
+            print exception
+        else:
+            results = tf.contrib.util.make_ndarray(result_future.result().outputs['outputs'])
+            for result in results[0]:
+                print ndarray_to_text(result)
```

Comment:
I've a question about this call.

Looking at how `output` is specified in the `model_exporter.init()` call it appears to be a list Tensors each converted from a SparseTensor. So I would guess that `results` should be a list of Tensors.

Thus the loop
```python
for result in results[0]:
   ...
```
doesn't make sense to me as it appears to be looping over a Tensor, the first Tensor in the results list which is the result of converting a SparseTensor to a Tensor; so, I wouldn't expect `result` to be the correct thing to be converting to a string.